src/lazy/binary/raw/v1_1/type_descriptor.rs

```diff
@@ -76,6 +76,10 @@ impl Opcode {
             (0xE, 0xA) => (NullNull, low_nibble, Some(IonType::Null)),
             (0xE, 0xB) => (TypedNull, low_nibble, Some(IonType::Null)),
             (0xE, 0xC..=0xD) => (Nop, low_nibble, None),
+            (0xF, 0x0) => (DelimitedContainerClose, low_nibble, None),
+            (0xF, 0x1) => (ListDelimited, low_nibble, Some(IonType::List)),
+            (0xF, 0x2) => (SExpressionDelimited, low_nibble, Some(IonType::SExp)),
+            (0xF, 0x3) => (StructDelimited, low_nibble, Some(IonType::Struct)),
```

Comment:
I really like this jump table. I've found myself wondering if we should add a higher-level `OpcodeKind` enum alongside the `OpcodeType` that captures some of the natural groupings that currently require testing for several `OpcodeType` variants. For example, having:
* `DelimitedContainer` for `0xF1`/`0xF2`/`0xF3`
* `EExp` for `0x00 .. 0x5F`
* etc

No `TODO` here, just thinking out loud.